CRM/CU/Upgrader.php

```diff
@@ -68,6 +68,16 @@ public function upgrade_4200() {
     return TRUE;
   } // */
 
+  public function upgrade_1100() {
+    $this->ctx->log->info('Applying update 1100 - Delete region custom field');
+    $regionId = civicrm_api3('CustomField', 'getvalue', array(
```

Comment:
@Edzelopez this is still deleting the field I thought we we were going to disable it

Reply:
Fixed